cmd/controller/app/server.go

```diff
@@ -73,6 +75,14 @@ func Run(cc *config.CompletedConfig, stopCh <-chan struct{}) error {
 		panic(annotatorController.Run(int(cc.AnnotatorConfig.ConcurrentSyncs), stopCh))
 	}
 
+	healthMux := http.NewServeMux()
+	healthz.InstallHandler(healthMux, healthz.NamedCheck("crane-controller", healthz.PingHealthz.Check))
```

Comment:
Using `crane-controller` may semantically conflict with the controller in the main project, so it is better to use `scheduler-controller` or `crane-scheduler-controller` here

Reply:
ok